boards/common/arduino-atmega/Makefile.dep

```diff
@@ -3,3 +3,5 @@ USEMODULE += boards_common_atmega
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
+
+include $(RIOTBOARD)/common/atmega/Makefile.dep
```

Comment:
Missing `\n` here. Maybe this is why Murdock fails to parse Makefiles?

Reply:
Last time I added `\n` gave me some additional new line errors-error?